server/server.js

```diff
@@ -25,7 +25,11 @@ if (!global.fetch) {
 
 const app = express();
 const server = http.createServer(app);
-server.port = process.env['NULLSTACK_SERVER_PORT'] || process.env['PORT'] || 5000;
+server.port = process.env['NULLSTACK_SERVER_PORT'] || process.env['PORT'];
```

Comment:
We should only force ports when !server.less as well, since hosts like vercel will not work with nullstack internal port anyway